size.go

```diff
@@ -103,7 +103,8 @@ func sizeOf(v reflect.Value, cache map[uintptr]bool) int {
 		reflect.Int, reflect.Uint,
 		reflect.Chan,
 		reflect.Uintptr,
-		reflect.Float32, reflect.Float64, reflect.Complex64, reflect.Complex128:
+		reflect.Float32, reflect.Float64, reflect.Complex64, reflect.Complex128,
+		reflect.Func:
```

Comment:
This is what I did locally to workaround #12. I was going to submit a PR, so thanks for doing it so quick!